libs/etterlatte-ktor/src/main/kotlin/ktor/ktorobo/DownstreamResourceClient.kt

```diff
@@ -124,8 +126,14 @@ class DownstreamResourceClient(
                         } else if (response.harContentType(ContentType.Text.Plain)) {
                             Ok(response.body<String>())
                         } else {
-                            logger.info("Mottok uhåndtert content-type: ${response.contentType()}")
-                            Ok(response.status)
+                            if (response.contentLength() == 0L) {
+                                logger.info("Mottok tom content type${response.contentType()}")
+                            } else {
+                                logger.warn(
+                                    "Mottok uhåndtert content-type: ${response.contentType()} lengde på innhold: ${response.contentLength()} ",
+                                )
+                            }
+                            Ok(null)
```

Comment:
Dette trengs for å kunne serialisere 2xx svar med tom body, for da er det ikke påkrevd å sette Content-Type.